### ISSUE TITLE
eyes_selenium is broken due to chunky_png-1.3.7 API change

### ISSUE THREAD

**Issue by bootstraponline** [applitools/Eyes.Selenium.Ruby#42]
I'm using `eyes_selenium 2.36.0` and it's failing to screenshot every time due to a `chunky_png` API change.

```
/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/chunky_png-1.3.7/lib/chunky_png/canvas/png_decoding.rb:120:in `decode_png_image_pass': wrong number of arguments (given 7, expected 6) (ArgumentError)
       	from /.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/chunky_png-1.3.7/lib/chunky_png/canvas/png_decoding.rb:120:in `decode_png_without_interlacing'
       	from /.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/chunky_png-1.3.7/lib/chunky_png/canvas/png_decoding.rb:99:in `decode_png_pixelstream'
       	from /.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/chunky_png-1.3.7/lib/chunky_png/canvas/png_decoding.rb:81:in `from_datastream'
       	from /.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/chunky_png-1.3.7/lib/chunky_png/image.rb:74:in `from_datastream'
       	from /.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/eyes_selenium-2.36.0/lib/applitools/utils/image_utils.rb:137:in `restore'
       	from /.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/eyes_selenium-2.36.0/lib/applitools/utils/image_utils.rb:90:in `block (2 levels) in stitch_images'
       	from /.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/eyes_selenium-2.36.0/lib/applitools/utils/image_utils.rb:78:in `each'
       	from /.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/eyes_selenium-2.36.0/lib/applitools/utils/image_utils.rb:78:in `block in stitch_images'
       	from /.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/eyes_selenium-2.36.0/lib/applitools/utils/image_utils.rb:77:in `tap'
       	from /.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/eyes_selenium-2.36.0/lib/applitools/utils/image_utils.rb:77:in `stitch_images'
       	from /.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/eyes_selenium-2.36.0/lib/applitools/method_tracer.rb:12:in `call'
       	from /.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/eyes_selenium-2.36.0/lib/applitools/method_tracer.rb:12:in `block in trace_method'
       	from /.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/eyes_selenium-2.36.0/lib/applitools/selenium/browser.rb:184:in `fullpage_screenshot'
       	from render.rb:39:in `<main>'
```

**Comment by danielputerman** [comment]
Fixed in 15d6908c3a578a107144a11d5e05152adf66a555 (locked chunky_png version to 1.3.6). Version 2.37.0 is now available.

Thanks for the feedback!

**Comment by bootstraponline** [comment]
Thanks!
